source/guides/guides/network-requests.md

```diff
@@ -318,6 +320,47 @@ cy.get('#results')
 - Response Body
 - Response Headers
 
+**Examples**
+
+```javascript
+cy.server()
+// spy on POST requests to /users endpoint
+cy.route('POST', '/users').as('new-user')
+// trigger network calls by manipulating web app's user interface, then
+cy.wait('@new-user')
+  .should('have.property', 'status', 201)
+// we can grab the XHR object again to run more assertions
+cy.wait('@new-user') // yields XHR object
```

Comment:
This will wait for an entirely new request/response to happen, so 2 POSTs. Did you mean to write `cy.get('@new-user')` here?

Reply:
ughh, you are right, thanks for catching this. I need to `cy.wait` once, but then use `cy.get` to retrieve it if I need it again. This is awesome doc for our users